.github/workflows/PR-4.x.yaml

```diff
@@ -21,5 +21,9 @@ jobs:
   macOS-X64:
     uses: opencv/ci-gha-workflow/.github/workflows/OCV-PR-4.x-macOS-x86_64.yaml@main
 
+  iOS:
+    if: "${{ contains(github.event.pull_request.labels.*.name, 'platform: ios/osx') }}"
```

Comment:
Do we really need this condition?
(BB-based CI always run iOS builds (in some reduced set))

Is there an option to move and handle that in workflows repository?

Reply:
What does "BB-based CI" mean?

Yes, there is an option, but if to compare with `khadas-vim3-tests` job, we run iOS workflow in CI repository, what means that we should have a large `if` condition there to handle all cases.

Reply:
Removed this line to run iOS Framework build on each commit in a PR.